plugins/org/api-report-alpha.md

```diff
@@ -0,0 +1,25 @@
+## API Report File for "@backstage/plugin-org"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { BackstagePlugin } from '@backstage/frontend-plugin-api';
+import { ExtensionDefinition } from '@backstage/frontend-plugin-api';
+import { ExternalRouteRef } from '@backstage/frontend-plugin-api';
+
+// @alpha (undocumented)
+const _default: BackstagePlugin<
+  {},
+  {
+    catalogIndex: ExternalRouteRef<undefined, false>;
+  }
+>;
+export default _default;
+
+// @alpha (undocumented)
+export const EntityGroupProfileCard: ExtensionDefinition<{
```

Comment:
No need to export the cards actually. Did you forget to run the api reports btw?